proposals/0357-alpenglow_validator_admission_ticket.md

```diff
@@ -0,0 +1,191 @@
+implemented by all Solana clients. The client specific data structure
+changes are omitted.
+
+## Dependencies
+
+The Validator Admission Ticket is specified in SIMD 326 (Alpenglow).
+
+The compressed BLS public key in Vote Account is specified in [SIMD 185 (Vote
+Account v4)](https://github.com/solana-foundation/solana-improvement-documents/pull/185)
+
+## Motivation
+
+Adding more staked validators to a blockchain does come with costs. At the
+very least, the votes and corresponding rewards need to be processed by
+every voting validator. Therefore, every additional staked validator increases
+the number of messages for every validator.
+
+Right now every voting validator pays voting fee on any vote transaction
+included in a block. The voting fee adds up to ~2 SOL per epoch if the
+validator votes most of the time. It is a burden, yet at the same time it's an
+economic barrier to having too many voting validators.
+
+VAT is only a temporary solution to maintain the current economic equilibrium.
+This proposal intentionally strives to keep voting validator protocol costs
+similar to pre-Alpenglow consensus.
+
+## New Terminology
+
+- **Validator Admission Ticket(VAT)**: The 1.6 SOL charged once per epoch to
+every validator eligible to participate in the next epoch
+
+## Different Vote Related Accounts
+
+Before Alpenglow, we have the following accounts related to voting:
+
+- Vote account for saving all the vote states and receiving commission
+
+- Identity account for receiving block rewards and priority fees, the vote
+transaction fees are currently paid out of this account
+
+After Alpenglow, we will still have roughly the same accounts serving the same
+purpose:
+
+- Vote account: This account must contain the correct BLS public key
+corresponding to vote authority keypair. It continues to keep all the vote
+credits and all validator identity/authority/commission information updates
+happen here, but the `votes` list in vote state will be empty. This will be
+the account where the 1.6 SOL VAT is deducted from.
+
+- Identity account: It continues receiving block rewards and priority fees.
+But we may change that in a separate SIMD.
+
+## Detailed Design
+
+### What is considered a valid vote account
+
+A valid vote account in an Alpenglow epoch must contain:
+
+- a BLS public key
+
+- at least 1.6 SOL VAT fee plus the necessary storage rent amount for a new
+epoch in the vote account
+
+When the staked validators for a new epoch are calculated, all validators must
+perform the following operations:
+
+- When there are more than 2,000 valid validators, sort all valid vote accounts
+by descending order of stake. If some validator with stake S is in position
+2001, then we remove all validators with stake S and less. If there are fewer
+than 2,000 valid validators, pick all of them.
+
+- Deduct 1.6 SOL VAT fee from each picked vote account once
+
+- Mark the fee burned and write the result into the bank
+
```

Comment:
these points seem unrelated to the heading?

Reply:
Oh, do you mean I should remove the text here and just leaves the "How to implement the checks"?